.github/workflows/docker-dev-build.yml

```diff
@@ -0,0 +1,87 @@
+name: Build Gladys dev images
+
+on:
+    workflow_dispatch:
+
+jobs:
+  build-front:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup nodejs
+        uses: actions/setup-node@v1
+        with:
+            node-version: 12.x
+      - run: cd front && npm install && npm run build
+      - name: Upload build artifact
+        uses: actions/upload-artifact@v1
+        with:
+          name: static
+          path: front/build
+  docker:
+    needs: build-front
+    name: Build and Push Gladys dev images
+    runs-on: ubuntu-latest
+    env:
+      # Export environment variables for all stages.
+      DOCKER_CLI_EXPERIMENTAL: enabled # for 'docker buildx'
+      DOCKERHUB_USER:  ${{secrets.DOCKERHUB_USER}}
+      DOCKERHUB_PASSWORD: ${{secrets.DOCKERHUB_PASSWORD}}
+      DOCKERHUB_REPO: gladys
+      DOCKER_PLATFORMS: >
+        linux/amd64
+        linux/arm/v7
+        linux/arm/v6
+        linux/arm64/v8
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Download build artifact
+      uses: actions/download-artifact@v1
+      with:
+        name: static
+    - name: Set up image tag
+      run: |
+          set -vx
+          echo "TAG=dev" >> $GITHUB_ENV
+          echo "DOCKER_BASE=${DOCKERHUB_USER}/${DOCKERHUB_REPO}" >> $GITHUB_ENV
```

Comment:
I just realized this won't work ! 

My dockerhub_user is my personal account but we push the image to the gladysassistant organization, so it can't be the same 

Maybe something like in CircleCI like REGISTRY/IMAGE ?